doc/ansible/dev/testing_guide.md

```diff
@@ -0,0 +1,219 @@
+# Testing Ansible Operators with Molecule
+
+## Getting started
+
+### Requirements
+To begin, you sould have:
```

Comment:
I think we should also include a note about the version of docker needed. According to https://github.com/moby/moby/pull/31352 it is in v17.05